crypto/sha/keccak1600.c

```diff
@@ -321,6 +343,149 @@ void KeccakF1600(uint64_t A[5][5])
     }
 }
 
+#elif defined(KECCAK_1X_ALT)
+/*
+ * This is variant of above KECCAK_1X that presses temporary storage
+ * even further, but at cost of more write references to A[5][5].
+ * It's less suitable if A[5][5] is memory bound, but better if it's
+ * register bound.
+ */
+
+static void Round(uint64_t A[5][5], size_t i)
+{
+    uint64_t C[5], D[5];
+
+    assert(i < (sizeof(iotas) / sizeof(iotas[0])));
```

Comment:
You know me, if it doesn't spare space [on screen] I tend to abstain from lingo :-)